docs/sanic/middleware.md

```diff
@@ -60,7 +60,7 @@ and the response will be returned. If this occurs to a request before the
 relevant user route handler is reached, the handler will never be called.
 Returning a response will also prevent any further middleware from running.
 
-```python
+```
```

Comment:
Why remove python ?

Reply:
Also weird that we're mixing md and rst

Reply:
Because it doesn't recognize the syntax as python because of the async/await language. It was creating a build error. Really these MD files need to be switched to RST. I was thinking of doing some documentation overhaul work after the LTS.

Reply:
Switching all the docs to rst is pretty high on my list too.

Reply:
Did you install the sphinx asyncio plugin when testing locally ? I think async/await should be supported. But we can take look into this later.

Reply:
I did not.

Reply:
I figured there must be a way to get it to understand async await... I just was punting it to another day so we don't need a dependency for it now.